histomicstk/utils/simple_mask.py

```diff
@@ -184,7 +184,6 @@ def estimate_variance(x, y, peak):
     SimpleMask
 
     """
-
```

Comment:
I realize this is a trivial change, but let's undo it to reduce the code difference.